app/views/claims/schools/show.html.erb

```diff
@@ -22,11 +22,11 @@
     <div class="govuk-grid-column-two-thirds">
       <%= render "shared/organisations/organisation_details", organisation: @school %>
       <h2 class="govuk-heading-m govuk-!-margin-top-9"><%= t(".additional_details") %></h2>
-      <%= render "additional_details", school: @school %>
+      <%= render "shared/organisations/school_additional_details", school: @school %>
```

Comment:
Hmm, should schools be nested within the organisations scope?

Just thinking out loud, if it was a shared partial between both providers and schools, I'd expect a `shared/organisations/additional_details` partial with an `organisation` local. As the schools and providers themselves, I'm thinking they might live at `shared/schools` and `shared/providers`, respectively. Although, we'll likely not need `shared/providers` for at least the Private Beta.

```suggestion
      <%= render "shared/schools/additional_details", school: @school %>
```